sdk/cloudmachine/Azure.Projects.Provisioning/api/Azure.Projects.Provisioning.net8.0.cs

```diff
@@ -10,6 +18,11 @@ public static partial class AzureProjectsCommands
     {
         public static bool TryExecuteCommand(this Azure.Projects.ProjectInfrastructure cmi, string[] args) { throw null; }
     }
+    public static partial class OfxExtensions
```

Comment:
what does Ofx stand for?

Reply:
opinionated framework. we will rename it. I don't even know whether that's how I want it to work (extension) or a subclass of project. I will make this decision when I finish implementing support for storing connections in app config.